sql/migrations/warehouse/000022_alter_wh_indexes.up.sql

```diff
@@ -0,0 +1,11 @@
+DROP INDEX IF EXISTS wh_staging_files_id_index;
+
+DROP INDEX IF EXISTS wh_staging_files_id_source_id_destination_id_index;
+
+DROP INDEX IF EXISTS wh_load_files_source_destination_id_table_name_index;
+
+DROP INDEX IF EXISTS wh_uploads_id_destination_id_namespace_status_index;
+
+DROP INDEX IF EXISTS wh_uploads_destination_id_namespace_index;
+
+DROP INDEX IF EXISTS wh_table_uploads_wh_upload_id_table_name_index;
```

Comment:
Can you provide some details for each index we drop

Reply:
Let me prepare a report on notion for each query we use, the query planner and the associated index used?

Reply:
https://www.notion.so/rudderstacks/Warehouse-Query-Index-Analysis-9f1ed261e0de478ead4bda05dced95be?pvs=4

Adding all the analysis over here. Added it for wh_uploads, Let me know if you need more information so that I can replicate it for all the other tables